src/main/java/gregtech/common/tileentities/machines/basic/MTEWorldAccelerator.java

```diff
@@ -120,12 +121,18 @@ public MTEWorldAccelerator(int pID, String pName, String pNameRegional, int pTie
     @Override
     public String[] getDescription() {
         return new String[] {
-            String
-                .format("Accelerating things (Max Radius: %d | Max Speed Bonus: x%d)", mTier, mAccelerateStatic[mTier]),
-            "Use a screwdriver to change mode, sneak to change Radius", "Use a wrench to change speed",
-            "To accelerate TileEntities, this machine has to be adjacent to it",
-            String.format("Normal mode consumes up to %s amperage, depending on radius", AMPERAGE_NORMAL),
-            String.format("TE mode consumes %s amperage", AMPERAGE_TE) };
+            "Machine Type: " + EnumChatFormatting.YELLOW + "World Accelerator, WA" + EnumChatFormatting.RESET,
+            "Max Speed Bonus " + EnumChatFormatting.GREEN
+                + String.format("x%d", mAccelerateStatic[mTier])
+                + EnumChatFormatting.RESET,
+            EnumChatFormatting.GOLD + "Blocks Mode"
+                + EnumChatFormatting.RESET
+                + String.format(": Radius 1-%d | Amps \u2264%s", mTier, AMPERAGE_NORMAL),
+            EnumChatFormatting.GOLD + "TileEntity Mode"
+                + EnumChatFormatting.RESET
+                + String.format(": Radius 1 | Amps \u2264%s", AMPERAGE_TE),
```

Comment:
The colons being a separate color is off and at odds with other tooltips
```suggestion
            EnumChatFormatting.GOLD + "Blocks Mode: "
                + EnumChatFormatting.RESET
                + String.format("Radius 1-%d | Amps \u2264%s", mTier, AMPERAGE_NORMAL),
            EnumChatFormatting.GOLD + "TileEntity Mode: "
                + EnumChatFormatting.RESET
                + String.format("Radius 1 | Amps \u2264%s", AMPERAGE_TE),
```